### PR TITLE
Add link to Clinger Cohen agency list; fixes #295

### LINKED ISSUE #295: Link to Clinger Cohen Act agency list?
Would it be reasonable to add a link to the list of Clinger Cohen Act agency acronyms? This is mentioned on the `agency` field on: https://code.gov/#/policy-guide/docs/compliance/inventory-code

and I believe is controlled here:

https://github.com/presidential-innovation-fellows/code-gov-web/blob/2804b87f62d2eff34ed486456cd079dae9713ff1/src/app/components/policy-guide/docs/compliance/compliance-inventory-code/compliance-inventory-code.template.html#L47

### PR DESCRIPTION
**Pull request by IanLee1521**
@jcastle -- This is what I was thinking with #295 , I just didn't have the right link. If this is acceptable to you, it would be nice to add the link from that webpage directly.

Cheers!

### CODE REVIEW

File: src/app/components/policy-guide/docs/compliance/compliance-inventory-code/compliance-inventory-code.template.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h4>Required</h4>
     <li><code>ifOther</code>: [string] A one- or two- sentence description of the measurement type used, if 'other' is selected as the value of 'method' field.</li>
 			</ul>
   </li>
-	<li><code>agency</code>: [string] The agency acronym for Clinger Cohen Act agency, as defined by the United States Government Manual. For example "GSA" or "DOD." Please use uppercase letters.</li>
+	<li><code>agency</code>: [string] The agency acronym for <a href="https://cfo.gov/about/">Clinger Cohen Act agency</a>, as defined by the United States Government Manual. For example "GSA" or "DOD." Please use uppercase letters.</li>
```
Comment (jcastle) [inline src/app/components/policy-guide/docs/compliance/compliance-inventory-code/compliance-inventory-code.template.html]: I would want to change this to say "The agency acronym for Clinger Cohen Act agency can be found through the CFO Act list, as CCA references the CFO Act. For example 'GSA' or 'DOD.' Please..."

Comment (IanLee1521) [reply src/app/components/policy-guide/docs/compliance/compliance-inventory-code/compliance-inventory-code.template.html]: Updated the wording.

**Comment by DanielJDufour** [comment]
Sorry. We got a bug in our build system.  Will probably take a couple days to sort it and then I'll merge the PR.

**Comment by IanLee1521** [comment]
No worries. Thanks @DanielJDufour and @jcastle !

**Comment by DanielJDufour** [comment]
@IanLee1521 , sorry, we still got an issue with our build.  Environmental variables that we use as a part of build and testing pipeline don't appear in builds initiated by forks. It's a security thing done by Circle CI.  I'm going to pull down these changes, push to GSA and then create a PR from there.

**Comment by DanielJDufour** [comment]
@IanLee1521 , I've moved your code over to an upstream branch.  Our CI process is weird at the moment and requires moving to upstream in order to pass through CI pipeline.  I'm closing this PR now and you can view your changes at https://github.com/GSA/code-gov-web/pull/444

**Comment by IanLee1521** [comment]
Thank you @DanielJDufour + @jcastle !

**Pull request closed without merging**